x-pack/solutions/security/packages/features/src/cases/v3_features/kibana_sub_features.ts

```diff
@@ -0,0 +1,206 @@
+    name: i18n.translate(
+      'securitySolutionPackages.features.featureRegistry.casesSettingsSubFeatureName',
+      {
+        defaultMessage: 'Case settings',
+      }
+    ),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            id: 'cases_settings',
+            name: i18n.translate(
+              'securitySolutionPackages.features.featureRegistry.casesSettingsSubFeatureDetails',
+              {
+                defaultMessage: 'Edit case settings',
+              }
+            ),
+            includeIn: 'all',
+            savedObject: {
+              all: [...savedObjects.files],
+              read: [...savedObjects.files],
+            },
+            cases: {
+              settings: [APP_ID],
+            },
+            ui: uiCapabilities.settings,
+          },
+        ],
+      },
+    ],
+  };
+
+  /* The below sub features were newly added in v2 (8.17) */
+
+  const casesAddCommentsCasesSubFeature: SubFeatureConfig = {
+    name: i18n.translate(
+      'securitySolutionPackages.features.featureRegistry.addCommentsSubFeatureName',
+      {
+        defaultMessage: 'Create comments & attachments',
+      }
+    ),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            api: apiTags.createComment,
+            id: 'create_comment',
+            name: i18n.translate(
+              'securitySolutionPackages.features.featureRegistry.addCommentsSubFeatureDetails',
+              {
+                defaultMessage: 'Add comments to cases',
+              }
+            ),
+            includeIn: 'all',
+            savedObject: {
+              all: [...savedObjects.files],
+              read: [...savedObjects.files],
+            },
+            cases: {
+              createComment: [APP_ID],
+            },
+            ui: uiCapabilities.createComment,
+          },
+        ],
+      },
+    ],
+  };
+  const casesreopenCaseSubFeature: SubFeatureConfig = {
+    name: i18n.translate(
+      'securitySolutionPackages.features.featureRegistry.reopenCaseSubFeatureName',
+      {
+        defaultMessage: 'Re-open',
+      }
+    ),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            id: 'case_reopen',
+            name: i18n.translate(
+              'securitySolutionPackages.features.featureRegistry.reopenCaseSubFeatureDetails',
+              {
+                defaultMessage: 'Re-open closed cases',
+              }
+            ),
+            includeIn: 'all',
+            savedObject: {
+              all: [],
+              read: [],
+            },
+            cases: {
+              reopenCase: [APP_ID],
+            },
+            ui: uiCapabilities.reopenCase,
+          },
+        ],
+      },
+    ],
+  };
+
+  const casesAssignUsersCasesSubFeature: SubFeatureConfig = {
```

Comment:
The above comment about this being in 8.17 doesn't apply here. Either delete that comment or add and 8.18 comment here